spec/QB-AddressbookSpec.js

```diff
@@ -0,0 +1,245 @@
+/* globals describe, beforeAll, expect, it */
+'use strict';
+
+describe('AddressBook', function() {
+  var REST_REQUESTS_TIMEOUT = 5000;
+
+  var isNodeEnv = typeof window === 'undefined' && typeof exports === 'object';
+
+  var QB = isNodeEnv ? require('../src/qbMain') : window.QB;
+
+  // var CREDS = isNodeEnv ? require('./config').CREDS : window.CREDS;
+  // var CONFIG =  isNodeEnv ? require('./config').CONFIG : window.CONFIG;
+  // var QBUser1 = isNodeEnv ? require('./config').QBUser1 : window.QBUser1;
+
+  var CREDS = {
+    'appId': 40771,
+    'authKey': 'a9VnGdCqXky7Bns',
+    'authSecret': 'aaznbzqmgsQq2jf'
+  };
+
+  var CONFIG = {
+    endpoints: {
+      api: 'apirc.quickblox.com', // set custom API endpoint
+      chat: 'chatrc.quickblox.com' // set custom Chat endpoint
+    },
+    chatProtocol: {
+      active: 2 // set 1 to use BOSH, set 2 to use WebSockets (default)
+    },
+    debug: { mode: 1 }
+  };
+
+  var QBUser1 = {
+    'id': 12747447,
+    'login': "js_jasmine22",
+    'password': "js_jasmine22",
+    'email': "js_jasmine22@quickblox.com"
+  };
+
+  var CONTACTS = [
+    {
+      'name':'Gordie Kann',
+      'phone': '1879108395',
+    },
+    {
+      'name':'Wildon Gilleon',
+      'phone': '2759108396'
+    },
+    {
+      'name':'Gaston Center',
+      'phone': '3759108396'
+    }
+  ];
+
+  var UID = 'A337E8A4-80AD-8ABA-9F5D-579EFF6BACAB';
+  var CONTACTS_BY_UID = [
```

Comment:
Need to rename CONTACTS_BY_UID to CONTACTS_BY_UDID (because udid is Universal Device ID)